frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -1,1080 +0,0 @@
-import { formatDate } from "../utils/date";
-import { TextWithTooltip } from "../commonComponents/TextWithTooltip";
-import {
-  TestCorrectionReason,
-  TestCorrectionReasons,
-} from "../testResults/viewResults/actionMenuModals/TestResultCorrectionModal";
-import MultiplexResultInputForm from "../testResults/MultiplexResultInputForm";
-import { MULTIPLEX_DISEASES } from "../testResults/constants";
-import { DeviceWithoutModelOrManufacturer } from "../utils/device";
-
-import { ALERT_CONTENT, QUEUE_NOTIFICATION_TYPES } from "./constants";
-import AskOnEntryTag, { areAnswersComplete } from "./AskOnEntryTag";
-import {
-  removeTimer,
-  TestTimerWidget,
-  updateTimer,
-  useTestTimer,
-} from "./TestTimer";
-import AoEModalForm from "./AoEForm/AoEModalForm";
-import "./QueueItem.scss";
-import { AoEAnswers } from "./AoEForm/AoEForm";
-import { QueueItemSubmitLoader } from "./QueueItemSubmitLoader";
-import { UPDATE_AOE } from "./addToQueue/AddToQueueSearch";
-
-const EARLIEST_TEST_DATE = new Date("01/01/2020 12:00:00 AM");
-
-interface AreYouSureProps {
-  isOpen: boolean;
-  cancelText: string;
-  continueText: string;
-  cancelHandler: () => void;
-  continueHandler: () => void;
-  children: React.ReactNode;
-}
-
-const AreYouSure: React.FC<AreYouSureProps> = ({
-  cancelHandler,
-  cancelText,
-  continueHandler,
-  continueText,
-  isOpen,
-  children,
-}) => (
-  <Modal
-    isOpen={isOpen}
-    style={{
-      content: {
-        maxHeight: "90vh",
-        width: "40em",
-        position: "initial",
-      },
-    }}
-    overlayClassName="prime-modal-overlay display-flex flex-align-center flex-justify-center"
-    contentLabel="Questions not answered"
-    ariaHideApp={process.env.NODE_ENV !== "test"}
-    onRequestClose={cancelHandler}
-  >
-    <div className="sr-modal-content">{children}</div>
-    <div className="margin-top-4 padding-top-205 border-top border-base-lighter margin-x-neg-205">
-      <div className="text-right prime-modal-buttons">
-        <Button onClick={cancelHandler} variant="unstyled" label={cancelText} />
-        <Button onClick={continueHandler} label={continueText} />
-      </div>
-    </div>
-  </Modal>
-);
-
-export const findResultByDiseaseName = (
-  results: MultiplexResultInput[],
-  name: string
-) =>
-  results.find((r: MultiplexResultInput) => r.diseaseName === name)
-    ?.testResult ?? null;
-
-export type QueriedTestOrder = NonNullable<
-  NonNullable<GetFacilityQueueQuery["queue"]>[number]
->;
-export type QueriedDeviceType = NonNullable<
-  GetFacilityQueueQuery["facility"]
->["deviceTypes"][number];
-export type QueriedFacility = GetFacilityQueueQuery["facility"];
-
-const convertFromMultiplexResponse = (
-  responseResult: QueriedTestOrder["results"]
-): MultiplexResultInput[] => {
-  return responseResult.map((result) => ({
-    diseaseName: result.disease?.name,
-    testResult: result.testResult,
-  }));
-};
-
-export type DevicesMap = Map<string, QueriedDeviceType>;
```

Comment:
⚠️  moved these types used in other files to a separate file `frontend/src/app/testQueue/TestCardForm/types.d.ts`